docs/src/proxy-configuration.md

```diff
@@ -18,21 +18,16 @@ properties:
       The configuration file version to use.
     enum:
       - v1alpha1
-  proxy:
-    type: object
-    description: |
-      Configuration of core proxy behavior.
-    properties:
-      id:
-        type: string
-        description: |
+  id:
+      type: string
+      description: |
           An identifier for the proxy instance.
-        default: On linux, the machine hostname is used as default. On all other platforms a UUID is generated for the proxy.
-      port:
-        type: integer
-        description: |
+      default: On linux, the machine hostname is used as default. On all other platforms a UUID is generated for the proxy.
+  port:
+      type: integer
+      description: |
           The listening port for the proxy.
```

Comment:
```suggestion
          The listening port. In "proxy" mode, the port for traffic to be sent to. In "manage" mode, the port to connect to the xDS API.
```

Something like that, since it's both?